.github/workflows/tests.yaml

```diff
@@ -452,21 +479,27 @@ jobs:
               uses: actions/upload-artifact@v4
               if: ${{ failure() && !env.ACT }}
               with:
-                  name: crash-core-darwin-${{ matrix.build_variant }}${{ matrix.chip_tool }}
+                  name:
+                      crash-core-darwin-${{ matrix.build_variant }}${{
+                      matrix.chip_tool }}
                   path: /cores/
                   # Cores are big; don't hold on to them too long.
                   retention-days: 5
             - name: Uploading diagnostic logs
               uses: actions/upload-artifact@v4
               if: ${{ failure() && !env.ACT }}
               with:
-                  name: crash-log-darwin-${{ matrix.build_variant }}${{ matrix.chip_tool }}
+                  name:
+                      crash-log-darwin-${{ matrix.build_variant }}${{
```

Comment:
could you please pull out restyle changes to a separate PR that is restyle only? This mixes restyle with functionality and that makes it harder to figure out the actual changes.

I am not convinced this restyle in particular is good - the newline change in the middle of a variable is hard to read. I am guessing it is some enforcement of 80-char line? Please don't do that if it affects readability.